.github/workflows/genericx86-64.yml

```diff
@@ -1,31 +1,62 @@
 name: Generic x86_64 (legacy MBR)
 
 on:
-  # With these triggers the Yocto jobs will run
-  # in parallel with the Flowzone jobs, which is fine for now
-  # and allows us to better control what we want to test and when.
-  # It is expected that Flowzone could fail, but yocto jobs will run.
+  # https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#onpushbranchestagsbranches-ignoretags-ignore
+  # https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet
   pull_request:
     branches:
-      - "main"
-      - "master"
+      - main
+      - master
+      # ESR branches glob pattern
+      - v20[0-9][0-9].[0-1]?[1470].[0-9]+
   pull_request_target:
     branches:
-      - "main"
-      - "master"
+      - main
+      - master
+      # ESR branches glob pattern
+      - v20[0-9][0-9].[0-1]?[1470].[0-9]+
   push:
+    branches:
+      - main
+      - master
+      # ESR branches glob pattern
+      - v20[0-9][0-9].[0-1]?[1470].[0-9]+
     tags:
+      # Semver tags glob pattern (includes ESR in format v20YY.MM.PATCH)
       - v[0-9]+.[0-9]+.[0-9]+\+?r?e?v?*
-      - v20[0-9][0-9].[0-1]?[1470].[0-9]+
   # Allows you to run this workflow manually from the Actions tab
   workflow_dispatch:
+    inputs:
+      deploy-s3:
+        description: Deploy artifacts to S3?
+        required: false
+        type: boolean
+        default: true
+      deploy-hostapp:
+        description: Deploy hostApp to balenaCloud?
+        required: false
+        type: boolean
+        default: true
+      deploy-esr:
+        description: Deploy as an ESR release?
+        required: false
+        type: boolean
+        default: false
+      deploy-ami:
+        description: Deploy AMI to AWS?
+        required: false
+        type: boolean
+        default: false
+      os-dev:
+        description: Enable OS development features
+        required: false
+        type: boolean
+        default: false
 
 jobs:
   yocto:
     name: Yocto
-    # FIXME: This workflow has dependencies on scripts in the balena-yocto-scripts repository
-    # which is pinned separately as a submodule in the device repo. Expect some drift but try to retain compatibility.
-    uses: balena-os/balena-yocto-scripts/.github/workflows/yocto-build-deploy.yml@v1.25.3
+    uses: balena-os/balena-yocto-scripts/.github/workflows/yocto-build-deploy.yml@7b65acc0280a2239d7f8fcfd7260151d97b6f3d5 # v1.25.9
```

Comment:
Would renovate be handling the update of the Yocto BTD workflow, I see you have update the workflow manually which is fine as this is the first PR. Getting clarication.

Reply:
Yes, Renovate is already opening PRs to bump the workflow version (the first commit of this PR is actually cherry-picked from the Renovate PR).

It's just the Renovate PR doesn't always merge for the usual reasons of Jenkins, etc.

Reply:
Yes, that's where my question. Why not commit the latest version here? `v1.25.9`

Reply:
Oversight, I didn't notice there was a newer version available.